conbench/api/benchmarks.py

```diff
@@ -113,10 +113,17 @@ def get(self):
             for benchmark_result in benchmark_results:
                 benchmark_result.z_score = 0
         elif batch_id:
-            benchmark_results = BenchmarkResult.all(batch_id=batch_id)
+            batch_id_list = batch_id.split(",")
+            benchmark_results = BenchmarkResult.search(
+                filters=[BenchmarkResult.batch_id.in_(batch_id_list)]
+            )
+            # benchmark_results = BenchmarkResult.all(batch_id=batch_id)
```

Comment:
Did you mean to leave this commented code?

Reply:
Oh thanks. No that was the old way we were doing this. I kept it in when I was testing. Better removed.